pyaedt/desktop.py

```diff
@@ -691,6 +624,113 @@ def disable_autosave(self):
         """
         self._main.oDesktop.EnableAutoSave(False)
 
+    def change_license_type(self, license_type="Pool"):
+        """Change the License Type between ``Pack`` and ``Pool``.
+
+        ..note: The command returns True even if the Key is wrong due to API limitation.
```

Comment:
This won't get rendered unless it follows this format
```suggestion
        .. note::
           The command returns True even if the Key is wrong due to API limitation.
```